docker/quarto/README.md

```diff
@@ -0,0 +1,53 @@
+# Quarto runner
+
+In this directory we define a dockerized environment to run Quarto. The image
+build on top of the VAST image, adding an R installation, Poetry, and of course
+Quarto. This makes it possible to call the VAST binary from within the
+notebooks.
```

Comment:
```suggestion
builds on top of the VAST image and adds R, Poetry, and Quarto. This creates a
runtime suitable for calling the VAST binary from notebooks.
```

Can we link the VAST image?

Reply:
the repository or the dockerfile?

Reply:
Whatever makes most sense. As a reader of this text, I was longing for an option to click when I read "VAST image", along the lines of "please take me to it".

Reply:
done